installer/conf/td-agent-bit-rs.conf

```diff
@@ -0,0 +1,29 @@
+[SERVICE]
+    Flush         30
+    Log_Level     info
+    Parsers_File  /etc/td-agent-bit/parsers.conf
+    Log_File      /var/opt/microsoft/docker-cimprov/log/fluent-bit.log
+
+[INPUT]
+    Name tail
+    Tag oms.container.log.telegraf.err.*
+    Path /var/opt/microsoft/docker-cimprov/log/telegraf.log
+    DB /var/opt/microsoft/docker-cimprov/state/telegraf-log-state.db
```

Comment:
Are u also adding the change to do the Exclude_Path for kube-system as part of this PR?